contract-tests/conftest.py

```diff
@@ -44,6 +44,10 @@ def requests_session(conf):
 def only_readonly(requests_session, conf):
     """Check if the current server is a readonly server, skip if it is not"""
     r = requests_session.get(conf.getoption("server") + "/__version__")
+
+    if r.status_code == 404:
```

Comment:
The version endpoint is an important part of Normandy, we should make it work in the environments that are under test instead of quietly ignoring it.

Reply:
Well, those tests always failed when I ran them locally so something needs to change with the current setup to make that __version__ end point work.